### PR TITLE
12 feature create nav

### PR DESCRIPTION
**Pull request by akajugz**

### CODE REVIEW

File: src/components/Nav/Nav.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.Nav {
+  &__links {
+    display: flex;
+    justify-content: space-between;
+    margin: 1.5rem;
+
+    .Nav__link {
```
Comment (Charlie-robin) [inline src/components/Nav/Nav.scss:7]: This is not BEM at the moment it is BEE as you have an element inside an element 

```scss
// BLOCK
.nav {
// ELEMENT
  &__links {
 // STYLES HERE ...
  }
// ELEMENT
  &__link {
     // STYLES HERE ...
    // MODIFIER 
    &:hover {
     // STYLES HERE ...
    }
  }
}
```

File: src/components/Nav/Nav.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import "./Nav.scss";
+import Home from "../../assets/icons/Home.svg";
+import Events from "../../assets/icons/Events.svg";
+import Calendar from "../../assets/icons/Calendar.svg";
+import About from "../../assets/icons/About.svg";
+
+const Nav = () => {
+  return (
+    <>
+      <nav className="Nav">
```
Comment (Charlie-robin) [inline src/components/Nav/Nav.tsx:11]: Will need to update class names here as well to lowercase `nav`

File: src/components/Nav/Nav.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { Link } from "react-router-dom";
+import "./Nav.scss";
+import Home from "../../assets/icons/Home.svg";
```
Comment (Charlie-robin) [inline src/components/Nav/Nav.tsx:3]: These are not components so should be lowercase hyphenated as they are files.
- BEM for styles
- PascalCasing for components
- camelCasing for ts
- lowercase-hypenated for other files e.g. images assets. 

🤯 

You will need to update the file names aswell as the imports e.g. `home.svg`

**Review by Charlie-robin — CHANGES_REQUESTED** [review]
Have a look at the comments you will need to resolve them to follow conventions

File: src/pages/Home/Home.tsx [hunk 4]
```diff
@@ -3,15 +3,7 @@ import Layout from "../../components/Layout/Layout";
...
-      <h1>Page Heading</h1>
-      <h2>Section Heading</h2>
-      <h3>Panel Heading</h3>
-      <p>
-        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Non error
-        dolor in nostrum minima odio a fuga saepe soluta adipisci perspiciatis
-        maxime tempora, temporibus beatae voluptas repellat rerum. Dignissimos,
-        necessitatibus.
-      </p>
+
```
Comment (olirob93) [inline src/pages/Home/Home.tsx:6]: If you remove this it gives an error in the console on my side. Can you put the dummy text back in?

File: src/components/Nav/Nav.tsx
Comment (olirob93) [inline src/components/Nav/Nav.tsx:1]: The nav looks good but it doesn't work as intended. Take a look at the NavLink component as part of the react router dom docs. You need to make sure when you navigate to a specific page the nav icon that correlates to the page is highlighted correctly.

https://reactrouter.com/en/main/components/nav-link

File: src/components/Nav/Nav.scss
Comment (olirob93) [inline src/components/Nav/Nav.scss:1]: you have styling for on hover but as it's a mobile app this doesn't show up. Take a look at my comment on the nav.tsx file

File: src/App.tsx [hunk 5]
```diff
@@ -1,12 +1,19 @@
...
+import Nav from "./components/Nav/Nav";
 
 const App = () => {
   return (
-    <Routes>
-      <Route path="*" element={<Home />} />
-    </Routes>
+    <>
+      <Routes>
+        <Route path="/" element={<Home />} />
```
Comment (olirob93) [inline src/App.tsx:10]: This is not going to be the main route / is going to be for the login page create another route for home for now

**Review by olirob93 — CHANGES_REQUESTED** [review]
see changes, let me know if any need clarifying

**Review by Charlie-robin — APPROVED** [review]
Looks good to me

**Review by olirob93 — APPROVED** [review]
looks good to me

**Pull request merged**